octobot_backtesting/channels_manager.py

```diff
@@ -24,14 +24,16 @@
 
 
 class ChannelsManager:
-    DEFAULT_REFRESH_TIMEOUT = 5
+    DEFAULT_REFRESH_TIMEOUT = 15
```

Comment:
It's the time before cancelling a backtesting iteration. With scripted lib and caches, it might take some time to finish the 1st iteration as it can be building cache values for the whole run (and then only read from this cache)

Reply:
So it can still be a problem depending on the computer power right?

Reply:
Right, computer power and the time to actually finish the evaluator cycle

Reply:
So how can we ensure that 15 is a good value?

Reply:
For now it's a value that is working rather well as it gives enough time to evaluate everything without being overkill. Considering we have no idea how complicated the scripts can be, it's complicated to figure out a dynamic value here. That's also why I added the `set_iteration_timeout` api in case a script need to change it